frontend/.eslintrc.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+	"env": {
+		"browser": true,
+		"es6": true
+	},
+	"extends": "eslint:recommended",
+	"globals": {
+		"Atomics": "readonly",
+		"SharedArrayBuffer": "readonly"
+	},
+	"parserOptions": {
+		"ecmaFeatures": {
+			"jsx": true
+		},
+		"ecmaVersion": 2018,
+		"sourceType": "module"
+	},
+	"plugins": [
+		"react"
+	],
+	"rules": {
+		"indent": [
+			"error",
+			"tab"

```

Comment:
Can you keep the default 4 space indentation instead please. The formatting shown in github is a nightmare to read.

Reply:
Sure. Already changed that :)